velox/expression/tests/SimpleFunctionInitTest.cpp

```diff
@@ -85,7 +85,7 @@ TEST_F(SimpleFunctionInitTest, initializationArray) {
       NonDefaultWithArrayInitFunction,
       Array<int32_t>,
       int32_t,
-      Array<int32_t>>({"non_default_behavior_with_init"});
+      Constant<Array<int32_t>>>({"non_default_behavior_with_init"});
```

Comment:
> why are we changing those? I see NonDefaultWithArrayInitFunction does handles non constant arrays

I thought the second argument is only handled in `initialize` function, so it should be marked as constant argument. But, yes, it can handle non constant arrays, I revert this change.

> question, what is the effect of marking an argument as constant in the signature.

Constant argument now is only checked in Fuzzer.  If one argument is marked as constant, only constant inputs are generated for it by Fuzzer.
https://github.com/facebookincubator/velox/issues/3995#issuecomment-1840285409